medical_appointment/tests/medical_appointment.yml

```diff
@@ -41,11 +41,11 @@
     email: arun@example.com

```

Comment:
File contains DOS line endings. Could you fix? This should do it for you in OS X:
```
perl -pe 's/\r\n|\n|\r/\n/g' medical_appointment.yml > medical_appointment.yml.fix && mv medical_appointment.yml.fix medical_appointment.yml
```